AmplifyPlugins/API/AWSAPICategoryPlugin/Interceptor/SubscriptionInterceptor/HeaderIAMSigningHelper.swift

```diff
@@ -0,0 +1,196 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSCore
+import AppSyncRealTimeClient
+
+struct HeaderIAMSigningHelper {
```

Comment:
nit: It'd be great to add some documentation about this class. Also, suggestion for naming it IAMHeaderSigningHelper

Reply:
Will handle this as a separate PR